layouts/shortcodes/skew.html

```diff
@@ -1,64 +1,92 @@
-<!-- capture version from first argument in "skew" -->
+{{/* capture version from first argument in "skew" */}}
 {{- $version := .Get 0 -}}
 
-<!-- strip "v" from latest verison  -->
+{{/* strip "v" from latest and current version  */}}
+{{- $currentVersion := site.Params.version -}}
+{{- $currentVersion := (replace $currentVersion "v" "") -}}
 {{- $latestVersion := site.Params.latest -}}
 {{- $latestVersion := (replace $latestVersion "v" "") -}}
 
-<!-- splits a string x.Xy into substrings separated by a "." delimiter  -->
-{{- $versionArray := split $latestVersion "." -}}
+{{/* splits a string x.Xy into substrings separated by a "." delimiter  */}}
+{{- $versionArrayCurrent := split $currentVersion "." -}}
 
-<!-- capture minorVersion ("Xy") in a variable -->
-{{- $minorVersion := int (index $versionArray 1) -}}
+{{/* capture minorVersion ("Xy") in a variable */}}
+{{- $minorVersionCurrent := int (index $versionArrayCurrent 1) -}}
 
-<!-- increment latest 1 minor number for "nextMinorVerison" -->
-{{- $nextMinorVersion := add $minorVersion 1 -}}
+{{/* splits a string x.Xy into substrings separated by a "." delimiter  */}}
+{{- $versionArrayLatest := split $latestVersion "." -}}
 
-<!-- subtract latest 1 minor number for "prevMinorVerison" -->
-{{- $prevMinorVersion := sub $minorVersion 1 -}}
+{{/* capture minorVersion ("Xy") in a variable */}}
+{{- $minorVersionLatest := int (index $versionArrayLatest 1) -}}
 
-<!-- subtract latest 2 minor numbers for "oldestMinorVerison" -->
-{{- $oldestMinorVersion := sub $minorVersion 2 -}}
+{{/* increment latest 1 minor number for "nextMinorVerison" */}}
+{{- $nextMinorVersionLatest := add $minorVersionLatest 1 -}}
 
-<!-- output nextMinorVersion based on captured arg -->
+{{/* subtract latest 1 minor number for "prevMinorVerison" */}}
+{{- $prevMinorVersionLatest := sub $minorVersionLatest 1 -}}
+
+{{/* subtract latest 2 minor numbers for "oldestMinorVerison" */}}
+{{- $oldestMinorVersionLatest := sub $minorVersionLatest 2 -}}
```

Comment:
maybe we should remove the prev* and oldest* shortcodes and use *VersionAddMinor everywhere.

Reply:
Eventually, perhaps. Right now we need to allow localizations to carry on using the existing shortcodes.